foundation/docs/blog/posts/2023-review.md

```diff
@@ -0,0 +1,158 @@
+---
+title: 2023 Recap
+date: 2024-01-05
+categories: 
+    - Foundation Updates
+authors:
+    - masonegger
+---
+
+2023 was a busy year for the PyTexas Foundation, from our annual conference, 
+launching new websites, and establishing a new virtual meetup the Foundation was 
+active all year.
+
+<!-- more -->
+
+## 2023
+
+### PyTexas 2023 Conference
+
+The 17th annual PyTexas Conference was held on April 1st - 2nd, 2023 in Austin, 
+Texas at the Austin Central Public Library. Brandon Rhodes and Andrew "Pandy" Knight 
+delivered the opening keynotes each morning respectively, and 17 speakers from 
+across the state and nation presented.
+
+We checked-in 141 attendees this year, 84% of attendees residing in Texas and 66% 
+being first time attendees. Various roles were represented amongst the attendees 
+but the majority of attendees this year were software engineers with 50% of attendees 
+having 7+ years of experience.
+
+At the end of the conference, we announced that PyTexas 2024 will be held at the same
+location April 19th - 21st, 2024.
+
+[View the PyTexas 2023 Website](https://archive.pytexas.org/2023){ .md-button .md-button--primary }
+
+### Launch of the PyTexas Foundation Websites and a Unified Tech Stack
+
+For many years the Board Members have discussed creating a PyTexas Foundation
+website. We also wanted to decide on a unified tech stack for all PyTexas assets.
+We wanted to consolidate deployments to a single platform, use static site generators
+whenever possible, and establish a unified set of tools and processes for building
+PyTexas assets. Being a low-revenue non-profit organization, anything that was
+free was preferred.
+
+We decided to go with [`mkdocs`](https://www.mkdocs.org/) and the 
+[Material](https://squidfunk.github.io/mkdocs-material/) extension for our static 
+site generator. The projects are open-source and built using Python. All static
+sites are hosted via GitHub pages using GitHub actions for CI. This year we launched
+the [PyTexas Foundation Website](https://pytexas.org), the [PyTexas Conference Website](https://pytexas.org/2024), and the [PyTexas Meetup Website](https://pytexas.org/meetup).
+
+A huge shout out to Laura Santamaria for her efforts in leading this transition.
+
+### Expansion of the PyTexas Community Discord
+
+In 2022 we launched a [Discord server](https://discord.gg/jNPAbcNukj) to replace
+our defunct Slack workspace for community engagement. With Slack announcing the
+limitation on message retention we decided the move was necessary. While we
+made the move in 2022, it wasn't until 2023 that we started to see people actually
+engaging in the server. We now host our monthly meetup there (more below), a weekly
+Friday hangout, and have various text channels for discussion. 
+
+[Join the PyTexas Discord](https://discord.gg/jNPAbcNukj){ .md-button .md-button--primary }
+
+### PyTexas Meetup
+
+After various discussions with Texas Pythonista's we discovered that Texas was missing
```

Comment:
Should the apostraphe come after the s: `Pythonistas'`?

Reply:
Wait would there be an apostrophe at all? Isn't it just plural?

Reply:
Correct. 🤦🏼

Reply:
Fixed